pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,18 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about a group version.
+type GroupVersionInfo struct {
+	GroupMeta latest.GroupMeta
+	// Info about the resources in this group version.
+	ResourcesStorageMap map[string]rest.Storage
+	// True, if this is the legacy core api ("/api").
+	IsLegacyCoreGroupVersion bool
+	// Optional: The group version to use for unversioned objects (like api.Status and unversioned.ListOptions).
+	// If nil, defaults to groupMeta.GroupVersion.
+	VersionForUnversionedObjects *unversioned.GroupVersion
```

Comment:
Renamed it to OptionsExternalVersion and also copied the same comment from apiserver.